packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -4459,4 +4459,37 @@ void main() {
     expect(errorDetails?.toString(), contains("InputDecorator's children reported a negative baseline"));
     expect(errorDetails?.toString(), contains('RenderStack'));
   });
+
```

Comment:
Add a comment here linking to the issue that this is testing:

```
// Regression test for: https://github.com/flutter/flutter/issues/75509
```

Reply:
Did you mean to put this comment on line 4493 instead?

Reply:
Oh sorry you got it right, it should be inside the `testWidgets` block 👍